verify-success.css

```
@@ -1,94 +1,134 @@
+  width: 100%;
+  max-width: 460px;
+  padding: 2.5rem;
+  border-radius: 20px;
+  background: rgba(15, 23, 42, 0.75);
+  border: 1px solid var(--line);
+  backdrop-filter: blur(14px);
+  box-shadow: 0 30px 80px rgba(2, 6, 23, 0.7);
+  animation: fadeUp 0.6s ease both;
+}
+
+@keyframes fadeUp {
```

Stylelint (17.4.0)

[error] Expected keyframe name "fadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)